src/pages/userProfilePage/components/editProfile.ts

```diff
@@ -1,128 +1,131 @@
-import { AddressesData, PersonalData } from "../../../types/interfaces/userProfilePage";
-import { createElement } from "../../../utils/abstract";
-import { updateData } from "../userProfilePageController";
+import { AddressesData, PersonalData } from '../../../types/interfaces/userProfilePage';
+import { createElement } from '../../../utils/abstract';
+import { updateData } from '../userProfilePageController';
 
 const legendsTextContent = {
   personal: 'Edit personal details',
   billing: 'Edit billing address',
-  shipping: 'Edit shipping address'
-}
+  shipping: 'Edit shipping address',
+};
 
+const createLabelElement = (): HTMLLabelElement => createElement('label', ['edit__label']);
 
-const createLabelElement = (): HTMLLabelElement => createElement('label', ['edit__label'])
+const createInputElement = (): HTMLInputElement => createElement('input', ['edit__input']);
```

Comment:
Optional. You are creating the same function in another file. It is better to place this function in separate module and export it from it. This will follow DRY